samples/retained-instance-store/src/main/java/com/badoo/ribs/samples/retained_instance_store/rib/RetainedInstancePresenter.kt

```diff
@@ -0,0 +1,37 @@
+package com.badoo.ribs.samples.retained_instance_store.rib
+
+import androidx.lifecycle.Lifecycle
+import com.badoo.ribs.android.subscribe
+import com.badoo.ribs.core.plugin.ViewAware
+import com.badoo.ribs.samples.retained_instance_store.utils.Clock
+
+interface RetainedInstancePresenter {
+
+    fun dispose()
```

Comment:
Just extend NodeLifecycleAware and override onDestroy instead

Reply:
Not required anymore